mitosheet/mitosheet/streamlit/v1/spreadsheet.py

```diff
@@ -70,6 +104,7 @@ def spreadsheet( # type: ignore
             sheet_functions: Optional[List[Callable]]=None, 
             importers: Optional[List[Callable]]=None, 
             df_names: Optional[List[str]]=None,
+            import_folder: Optional[str]=None,
```

Comment:
Since this is a big vulnerability, maybe we should not make it an optional parameter, but instead should make users specifically turn it off. Otherwise, we should ensure that the default cause should be safe.

Reply:
Ooo well I see the feature is turned off in streamlit if you don't set the import_folder so that seems good!